site/get-started.md

```diff
@@ -4,14 +4,14 @@ menu_order: 10
 ---
 
 - [Get started with Flux](#get-started-with-flux)
-  * [Prerequisites](#prerequisites)
-    + [A Note on GKE with RBAC enabled](#a-note-on-gke-with-rbac-enabled)
-  * [Set up Flux](#set-up-flux)
-  * [Deploying Flux to the cluster](#deploying-flux-to-the-cluster)
-  * [Giving write access](#giving-write-access)
-  * [Committing a small change](#committing-a-small-change)
-  * [Confirm the change landed](#confirm-the-change-landed)
-  * [Conclusion](#conclusion)
+  - [Prerequisites](#prerequisites)
```

Comment:
Is this invalid?

https://www.markdownguide.org/basic-syntax/#unordered-lists
https://docs.gitlab.com/ee/user/markdown.html#lists
https://daringfireball.net/projects/markdown/syntax#list

Reply:
I think it was https://github.com/DavidAnson/markdownlint/blob/v0.11.0/doc/Rules.md#md004---unordered-list-style that showed up for me. I can roll back the change too.

Reply:
I think that is because the default configuration is not set to 'sublist' style.

> For example, the following is valid for the sublist style because the outer-most indent uses asterisk, the middle indent uses plus, and the inner-most indent uses dash:
```
* Item 1
  + Item 2
    - Item 3
  + Item 4
* Item 4
  + Item 5
```

I have no strong opinion on this but I think it improves readability for raw  markdown files (which I read more often than parsed files).